plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -1477,14 +1493,17 @@ public void pull(final PullParams params, final ProgressMonitor progressMonitor)
     private void pull(final PullParams params,
                       final ProgressMonitor progressMonitor,
                       final URI dockerDaemonUri) throws IOException, InterruptedException {
-        final String image = params.getImage();
         final String registry = params.getRegistry();
+        final AuthConfigs authConfigs = params.getAuthConfigs();
 
         try (DockerConnection connection = connectionFactory.openConnection(dockerDaemonUri)
                                                             .method("POST")
                                                             .path("/images/create")
-                                                            .query("fromImage", registry != null ? registry + '/' + image : image)
-                                                            .header("X-Registry-Auth", initialAuthConfig.getAuthConfigHeader())) {
+                                                            .query("fromImage", params.getImageFqn())
```

Comment:
I used general meaning of FQN. May be it is not good idea because tag is excluded. `fullImagePath` WDYT?